polybar-scripts/xkcd-notifier/README.md

```diff
@@ -0,0 +1,25 @@
+# Script: xkcd-notifier
```

Comment:
Lets rename the script, directory and other descriptions to `notification-xkcd`.